.travis.yml

```diff
@@ -0,0 +1,25 @@
+dist: xenial
+sudo: required
+
+language: generic
+
+addons:
+  apt:
+    packages:
+     - build-essential
+     - make
+     - perl
+     - git
+     - unzip
+     - m4
+     - gcc
+     - curl
+     - libpcre3
+     - libpcre3-dev
+     - zlib1g-dev
+
+jobs:
+  include:
+    # Known builds we want to succeed
+    - script: ./kong-ngx-build --prefix prefix --openresty 1.11.2.5 --openssl 1.0.2n --luarocks 2.4.3 --no-openresty-patches
+    - script: ./kong-ngx-build --prefix prefix --openresty 1.13.6.2 --openssl 1.1.1c --luarocks 3.1.3
```

Comment:
I wouldn't be against some test cases for the version comparison utilities... Especially considering the tricky cases such as letters in OpenSSL version schemes and 4 digits versioning schemes for OpenResty. But ultimately not a blocker atm if you don't have the time.

Reply:
If we add a check on `basename $0` before calling the main function, I think ultimately we could source the script and test the version functions separately.